Makefile

```diff
@@ -30,6 +30,10 @@ EXTRA_GOFLAGS ?=
 
 MAKE_VERSION := $(shell $(MAKE) -v | head -n 1)
 
+ifneq ($(RACE_ENABLED),)
+	GOTESTFLAGS ?= -race
+endif
+
```

Comment:
What does it mean?

Reply:
@bagasme -> https://blog.golang.org/race-detector